packages/wallet/src/redux/protocols/concluding/responder/reducer.ts

```diff
@@ -170,6 +170,7 @@ function concludeApproved(protocolState: NonTerminalCState, sharedData: Storage)
       false,
       lastestCommitment.allocation,
       lastestCommitment.destination,
+      makeLocator(CONSENSUS_UPDATE_PROTOCOL_LOCATOR),
```

Comment:
This is counter-intuitive. Is `CONSENSUS_UPDATE_PROTOCOL_LOCATOR` not a protocol locator?

Reply:
I'll update this. I most likely switched to `makeLocator` before I realized `concluding` is a top level protocol and I thought there would be another `protocolLocator` to pass in.

Reply:
Ah, good point. In that case, the protocolLocator should be `[]`